flask_restx/utils.py

```diff
@@ -122,3 +137,38 @@ def unpack(response, default_code=HTTPStatus.OK):
         return data, code or default_code, headers
     else:
         raise ValueError("Too many response values")
+
+
+def parse_rule(rule: str) -> t.Iterator[t.Tuple[t.Optional[str], t.Optional[str], str]]:
```

Comment:
Function entirely removed from `werkzeug` on newer versions

Reply:
[It does](https://github.com/pallets/werkzeug/blob/2.2.x/src/werkzeug/routing/rules.py#L584) but it seemed more cumbersome than [the initial](https://github.com/pallets/werkzeug/blob/2.0.x/src/werkzeug/routing.py#L215) function.

Gonna be honest: haven't looked into this new module much as I found importing the function faster than understanding the new logic and adapting (with the possibility of it not being equivalent). If needed be, I can do that,